tests/unit/storage_rocks.cpp

```diff
@@ -9,111 +9,109 @@
-// ASSERT_EQ(memgraph::utils::SerializeVertex(*vertex.vertex_), "|" + memgraph::utils::SerializeIdType(gid));
-// }
+  ~RocksDBStorageTest() override {}
 
-/// Serialize vertex with gid and its single label.
-// TEST_F(RocksDBStorageTest, SerializeVertexGIDLabels) {
-//   auto acc = storage->Access();
-//   auto vertex = acc->CreateVertex();
-//   auto ser_player_label = acc->NameToLabel("Player");
-//   auto player_result = vertex.AddLabel(ser_player_label);
-//   auto gid = vertex.Gid();
-//   ASSERT_EQ(memgraph::utils::SerializeVertex(*vertex.vertex_),
-//             std::to_string(ser_player_label.AsInt()) + "|" + memgraph::utils::SerializeIdType(gid));
-// }
+ protected:
+  std::unique_ptr<memgraph::storage::Storage> storage;
+  memgraph::storage::Config config_;
+};
 
-// /// Serialize vertex with gid and its multiple labels.
-// TEST_F(RocksDBStorageTest, SerializeVertexGIDMultipleLabels) {
-//   auto acc = storage->Access();
-//   auto vertex = acc->CreateVertex();
-//   auto ser_player_label = acc->NameToLabel("Player");
-//   auto ser_person_label = acc->NameToLabel("Person");
-//   auto ser_ball_label = acc->NameToLabel("Ball");
-//   // NOLINTNEXTLINE
-//   auto player_res = vertex.AddLabel(ser_player_label);
-//   auto person_res = vertex.AddLabel(ser_person_label);
-//   auto ball_res = vertex.AddLabel(ser_ball_label);
-//   auto gid = vertex.Gid();
-//   ASSERT_EQ(memgraph::utils::SerializeVertex(*vertex.vertex_),
-//             std::to_string(ser_player_label.AsInt()) + "," + std::to_string(ser_person_label.AsInt()) + "," +
-//                 std::to_string(ser_ball_label.AsInt()) + "|" + memgraph::utils::SerializeIdType(gid));
-// }
+TEST_F(RocksDBStorageTest, SerializeVertexGID) {
+  auto acc = storage->Access();
+  auto vertex = acc->CreateVertex();
+  auto gid = vertex.Gid();
+  ASSERT_EQ(memgraph::utils::SerializeVertex(*vertex.vertex_), "|" + memgraph::utils::SerializeIdType(gid));
+}
 
-// /// Serialize edge.
-// TEST_F(RocksDBStorageTest, SerializeEdge) {
-//   auto acc = storage->Access();
-//   auto vertex1 = acc->CreateVertex();
-//   auto vertex2 = acc->CreateVertex();
-//   auto edge = acc->CreateEdge(&vertex1, &vertex2, acc->NameToEdgeType("KNOWS"));
-//   auto gid = edge->Gid();
-//   auto ser_result = memgraph::utils::SerializeEdge(vertex1.Gid(), vertex2.Gid(), edge->EdgeType(), edge->edge_.ptr);
-//   ASSERT_EQ(ser_result.first,
-//             memgraph::utils::SerializeIdType(vertex1.Gid()) + "|" + memgraph::utils::SerializeIdType(vertex2.Gid()) +
-//                 "|0|" + std::to_string(edge->EdgeType().AsInt()) + "|" + memgraph::utils::SerializeIdType(gid));
-//   ASSERT_EQ(ser_result.second,
-//             memgraph::utils::SerializeIdType(vertex2.Gid()) + "|" + memgraph::utils::SerializeIdType(vertex1.Gid()) +
-//                 "|1|" + std::to_string(edge->EdgeType().AsInt()) + "|" + memgraph::utils::SerializeIdType(gid));
-// }
+TEST_F(RocksDBStorageTest, SerializeVertexGIDLabels) {
```

Comment:
Should we also add tests for `SerializeVertexGIDLabelsOneProperty` and `SerializeVertexGIDLabelsManyProperties` ?

Reply:
I mean we can but I don't see what we would get more since properties are serialized as value and labels as key so I think that we wouldn't get nothing more than we have.

Reply:
We will be sure that properties are listed through commas as we designed. If something will be changed in serialization function we will catch it.

Reply:
but properties are serialized just as with in-memory, no commas there?

Reply:
Can you explain what do you mean?

Reply:
Oops my bad. I messed with properties and labels. I meant to add multiple labels, not properties. You are right, properties are put as a serialized buffer.